spec/requests/subject_spec.rb

```diff
@@ -231,113 +231,189 @@
       context "when requesting data for someone else" do
         it "renders the expected page" do
           expect(response).to render_template(:edit)
-          expect(response.body).to include("Upload their ID")
-          expect(response.body).to include("For example, a copy of their driving licence or passport")
-          expect(response.body).to include("For example an electricity or council tax bill in their name")
+          expect(response.body).to include("Upload their photo ID")
+          expect(response.body).to include("For example, a driving licence or passport")
         end
       end
 
       context "when requesting data for self" do
-        let(:information_request) { build(:information_request_for_self) }
+        let(:photo_upload) { create(:attachment) }
+        let(:information_request) { build(:information_request_by_friend, subject_photo_id: photo_upload.id) }
```

Comment:
The test title is "data for self" but the fixture used is `:information_request_by_friend`